l10n_it_fatturapa_out_rc/views/invoice_it_template.xml

```diff
@@ -50,17 +50,6 @@
                 t-esc="record.fiscal_document_type_id.code"
             />
             </xpath>
-            <xpath expr="//DettaglioPagamento/ImportoPagamento" position="replace" />
-            <xpath expr="//DettaglioPagamento/ImportoPagamento" position="replace">
-                <ImportoPagamento
-                t-if="record.company_id.xml_divisa_value == 'keep_orig'"
```

Comment:
Questa parte di gestione relativa di `xml_divisa_value` dove viene fatta? Non serve più?

Reply:
Viene fatta qui:
https://github.com/OCA/l10n-italy/blob/e187808d1868dfba11483ed7237345b3fa69460a/l10n_it_fatturapa_out/models/company.py#L29-L36

https://github.com/OCA/l10n-italy/blob/e187808d1868dfba11483ed7237345b3fa69460a/l10n_it_fatturapa_out/data/invoice_it_template.xml#L555-L562

https://github.com/OCA/l10n-italy/blob/e187808d1868dfba11483ed7237345b3fa69460a/l10n_it_fatturapa_out/data/invoice_it_template.xml#L579-L586

Semplicemente adesso gli altri moduli (come `l10n_it_fatturapa_out_rc` in questo caso) invece di intervenire in quei punti sostituendo gli element dell'XML, intervengono a monte sulle funzioni che forniscono quei valori, lasciando l'XML del template intatto.

Reply:
Ok, grazie del chiarimento!